packages/slate-react/src/components/editable.tsx

```diff
@@ -186,10 +186,14 @@ export const Editable = (props: EditableProps) => {
     if (newDomRange) {
       domSelection.addRange(newDomRange!)
       const leafEl = newDomRange.startContainer.parentElement!
+      leafEl.getBoundingClientRect = newDomRange.getBoundingClientRect.bind(
+        newDomRange
+      )
```

Comment:
when testing this fix, did you scroll around a lot? Worried about performance issues when building and destroying this every time you scroll

Reply:
I haven't found any performance issues so far. If that was the case however I think we would see the issue on typing rather than on scrolling given this piece of code is only called when the selection changes, not the scroll. I believe this should be a cheap operation either way. Or at least probably cheaper than the DOM access operations that happen on `toDOMRange` sequential to this.